content/docs/fairing/azure/az-kubeflow-notebook.md

```diff
@@ -0,0 +1,186 @@
+* You must have the `kubectl` command line interface installed and configured
+  to use the Kubernetes cluster where Kubeflow is installed. In the preceding
+  installation guide, the command `az aks get-credentials` configures `kubectl`
+  to access your Kubernetes cluster.
+
+## Create Azure Container Registry and Storage
+
+In this example notebook, Azure Container Registry (ACR) is used to host docker
+images for deployment and Azure Storage (Storage) is used as build context for
+in-cluster building.
+
+You can re-use existing ACR and Storage or create new ones. For more
+information, see the documentation for [ACR][az-container-reg] and
+[Storage][az-storage].
+
+After you have created your ACR and Storage resources, you must configure a
+Service Principal with access to these resources. If you want to use the same
+Service Principal as your AKS cluster, you can get the Service Principal ID
+with the following command:
+
+```bash
+az aks show -g <resource-group-name> -n <name>
+```
+
+The preceding command has output like:
+
+```
+  "servicePrincipalProfile": {
+    "clientId": "<id>"
+  },
+```
+
+The role you grant must have at least read and write permissions to ACR and
+Storage.
+
+To learn more about how to grant the Service Principal access, follow the
+[Azure role-based authentication documentation][az-roles].
+
+## Setting up credentials as Kubernetes secrets
+
+Before running the notebook, you must first configure credentials so that the
+Python code running within the cluster can access the Azure resources required
+to train and deploy a model.
+
+Run the following commands to set up your credentials as a Kubernetes secret.
+
+1. Set environment variables to use in the following commands.
+
+   ```bash
+   export AZ_CLIENT_ID=<service-principal-client-id>
+   export AZ_CLIENT_SECRET=<service-principal-client-secret>
+   export AZ_TENANT_ID=<tenant-id>
+   export AZ_SUBSCRIPTION_ID=<subscription-id>
+   export TARGET_NAMESPACE=<target-namespace e.g. kubeflow-anonymous>
+   export ACR_NAME=<acr-name>
+   ```
+
+   * **AZ_CLIENT_ID:** The Service Principal client ID. You can get the
+     `client_id` property from ~/.azure/aksServicePrincipal.json.
+   * **AZ_CLIENT_SECRET:** The Service Principal secret. You can get the
+     `client_secret` property from ~/.azure/aksServicePrincipal.json.
+   * **AZ_TENANT_ID:** The Azure Tenant ID of your account. You can get the
+     Tenant ID from the `tenantId` field in the output of `az account show`.
+   * **AZ_SUBSCRIPTION:** The Azure Subscription ID of your account. You can
+     get the Subscription ID from the `id` field in the output of `az account
+     show`.
+   * **TARGET_NAMESPACE:** Specify the namespace that your Notebook Server is
+     in. For example, this guide recommends using `kubeflow-anonymous`.
+   * **ACR_NAME:** The name of an ACR that the Service Principal can access.
+
+1. Use the following command to create a secret that Kubeflow can use to access
```

Comment:
The list formatting is broken here and below - see preview:
https://deploy-preview-1478--competent-brattain-de2d6d.netlify.com/docs/fairing/azure/az-kubeflow-notebook/#setting-up-credentials-as-kubernetes-secrets

To fix, try removing the language indicator "`bash`" from the code block above. If that doesn't work then try adjusting the indentation.

Reply:
Ack! Sorry for not noticing that!